mls.json

```diff
@@ -1,4 +1,12 @@
 {
+    "public-hydra-logs@w3.org": {
+        "HydraCG/Specifications": {
+            "events": ["issues.opened", "issues.closed", "issues.reopened", "issue_comment.created", "issue_comment.edited", "pull_request.opened", "pull_request.closed", "pull_request_review.submitted", "pull_request_review_comment.created", "pull_request_review_comment.edited", "commit_comment.created", "push"],
+            "branches": {
```

Comment:
@lanthaler github-notify-ml doesn't support the following events at the moment:
"issues.reopened", "pull_request_review.submitted", "pull_request_review_comment.created", "pull_request_review_comment.edited", "commit_comment.created" 

Can you amend accordingly?

To add support for more events, issues (or event pull requests) on the [repo of the code](https://github.com/dontcallmedom/github-notify-ml/) are welcomed :)